src/stylesheets/mixins/_font-smoothing.scss

```diff
@@ -0,0 +1,11 @@
+@mixin font-smoothing-light-on-dark() {
+    // on
+    -webkit-font-smoothing : antialiased;
+    -moz-osx-font-smoothing: grayscale;
+}
+
+@mixin font-smoothing-dark-on-light() {
```

Comment:
This mixin shouldn't be necessary if you're only applying font smoothing where you need it.